testsuite/MDAnalysisTests/topology/test_crd.py

```diff
@@ -35,6 +36,7 @@ class TestCRDParser(ParserBase):
                       'resids', 'resnames', 'resnums',
                       'segids']
     guessed_attrs = ['masses', 'types']
+
     expected_n_atoms = 3341
     expected_n_residues = 214
-    expected_n_segments = 1
+    expected_n_segments = 1
```

Comment:
```suggestion
    expected_n_segments = 1

    def test_guessed_masses(self, filename):
        u = mda.Universe(filename)
        expected = [14.007,  1.008,  1.008,  1.008, 12.011,  1.008, 12.011]
        assert_allclose(u.atoms.masses[:7], expected)
    
    def test_guessed_types(self, filename):
        u = mda.Universe(filename)
        expected = ['N', 'H', 'H', 'H', 'C', 'H', 'C']
        assert (u.atoms.types[:7] == expected).all()
```

Suggesting to add a newline and tests to check behaviour hasn't changed